crates/bevy_app/src/app.rs

```diff
@@ -1007,6 +1007,7 @@ impl App {
 }
 
 fn run_once(mut app: App) {
+    warn!("No app runner detected. defaulting to `run_once`. If this is intentional, consider using `App::update` directly instead.");
```

Comment:
This isn't compiling due to the missing `trace` cfg annotation.

I'd like to branch on that feature flag here, and attempt to warn (maybe just using println?) even if that feature is off.

Users will commonly hit this when *all* features are off, so it's important to have a fallback.

Reply:
I'm not sure I follow. Is adding the `bevy_utils::tracing::warn` import in 48e0407 not enough?